server/src/test/java/org/elasticsearch/action/admin/cluster/stats/CCSTelemetrySnapshotTests.java

```diff
@@ -33,7 +33,7 @@ public class CCSTelemetrySnapshotTests extends AbstractWireSerializingTestCase<C
 
     private LongMetricValue randomLongMetricValue() {
         LongMetric v = new LongMetric();
-        for (int i = 0; i < randomIntBetween(1, 10); i++) {
+        for (int i = 0; i < randomIntBetween(5, 10); i++) {
```

Comment:
It's not immediately obvious what issue we're looking to fix and how this change fixes it. 

Can you please expand in the PR description to observed problem? 

I suspect the problematic usage of this random long method lies in `mutateInstance` ? (based on the link issue) If this is correct I think we should use `randomValueOtherThan()` to make sure `mutateInstance` generates a value that's different than the input.

Reply:
@andreidan I changed the code to use `randomValueOtherThan`